utils/networking/Cargo.toml

```diff
@@ -15,8 +15,7 @@ lazy_static = "1.4"
 log = "0.4"
 regex = "1"
 
-#ya-relay-stack = { version = "0.4.1", optional = true }
-ya-relay-stack = { git = "https://github.com/golemfactory/ya-relay.git", rev = "f9371b572204365920bbe1f6cc9b3e757bf12887", optional = true }
+ya-relay-stack = { version = "0.4.1", optional = true }
```

Comment:
Using specific commit version is unnecessary when specific commit is in `[patch.crates-io]` of top level `Cargo.toml`.